lint/rule_test.go

```diff
@@ -294,13 +293,16 @@ func TestRuleIsEnabled(t *testing.T) {
 	// Create appropriate test permutations.
 	tests := []struct {
 		testName       string
+		ruleName       RuleName
 		fileComment    string
 		messageComment string
 		enabled        bool
 	}{
-		{"Enabled", "", "", true},
-		{"FileDisabled", "api-linter: test=disabled", "", false},
-		{"MessageDisabled", "", "api-linter: test=disabled", false},
+		{"Enabled", NewRuleName("test"), "", "", true},
+		{"FileDisabled", NewRuleName("test"), "api-linter: test=disabled", "", false},
+		{"MessageDisabled", NewRuleName("test"), "", "api-linter: test=disabled", false},
+		{"NameNotMatch", NewRuleName("test"), "", "api-linter: other=disabled", true},
+		{"AliasDisabled", NewRuleName("core", "0140", "lower-snake"), "", "api-linter: naming-format=disabled", false},
```

Comment:
I don't think this test should depend on the actual rule implementations that we have in the repo, because those will change (i.e. we will add / remove aliases as we fix the usages of them). Maybe inject the aliasMap to ruleIsEnabled and then make a dummy one for the test. I think there's only one non-test usage of ruleIsEnabled so it should be an easy fix.

Reply:
we can also create an aliases.go that holds a global alias map and then have [the linter usage](https://github.com/googleapis/api-linter/blob/master/lint/lint.go#L82) pass that one in. Then the aliases aren't buried in code.

Reply:
Done. Thanks!